shell.nix

```diff
@@ -38,7 +38,8 @@ lib.overrideDerivation postgrest.env (
 
     shellHook =
       ''
-        complete -c postgrest-watch
+        source ${pkgs.bashCompletion}/etc/profile.d/bash_completion.sh
+        complete -F _command postgrest-watch
```

Comment:
@monacoremo Your first take at this was better. `-c` did not allow to complete filenames after `postgrest-watch postgrest-run` properly anymore, but `_command` does.

I think it only failed on my system, because my host doesn't have `bash-completion` installed. I use zsh. Adding `bash-completion` inside `nix-shell` fixes the `_command not found`.

Reply:
Awesome, had no idea about the bashCompletion package - great solution!